Dynatrace/dynatrace-operator
# Remove unnecessary dependencies from operator-image

# Description
 
As we copy the whole util-linux package over to the operator image, it has things included which are not needed and cause vulnerabilities. As we only want to add tar, mount and umount it is sufficient to just copy them over one by one and not the whole folder.

## How can this be tested?
Deploy the operator and check if everything is working as expected (injection, mounting,...)
Additionally to verify that the `kubectl cp` command works, try to copy a file from the operator pod on your local machine.


## Checklist
